RunGoogleClassroom.py

```diff
@@ -36,21 +36,15 @@ def Parse_Args(filter1):
     parser.set_defaults(output=True)
     parser.set_defaults(file=True)
     args = parser.parse_args()
-    parsed = True
-    if (args.xlsx == None):
-        print("Missing required argument (they're all required)")
-        parser.print_help()
-        parsed = False
-    else:
-        filter1.Read_Args(args)    
-    return (parsed, args.assignment, args.xlsx, args.output, args.file, args.golden_source)
+    filter1.Read_Args(args)
+    return args
```

Comment:
By returning the args object, we can refer to members by name instead of relying on their position within a tuple.